pkgs/desktops/gnome-3/extensions/tilingnome/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, glib }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-extension-tilingnome-${version}";
+  version = "2019-01-18";
```

```suggestion
  version = "unstable-2019-01-18";
```
https://nixos.org/nixpkgs/manual/#sec-package-naming

---

This package does not have releases; there isn't much use in differentiating it from a stable release

---

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

---

Hm, reading it over again I might have been wrong with the diff and it should be set in the package name instead, as in `gnome-shell-extension-tilingnome-unstable`. Doesn't matter much though.